docs/BR.md

```diff
@@ -391,6 +396,10 @@ The Definitions found in the CA/Browser Forum's Network and Certificate System S
 
 **P-Label**: A XN-Label that contains valid output of the Punycode algorithm (as defined in RFC 3492, Section 6.3) from the fifth and subsequent positions.
 
+**Primary CAA Determination**: The basis for which the CA has checked CAA records and determined its authority to issue a Certificate for the requested domain(s).
+
+**Primary Domain Validation Determination**: The basis for which the CA has determined Applicant authority and/or domain authorization or control using one of the methods described in Section 3.2.2.4 of this Policy.
```

Comment:
Reading the changes further down it looks like the intention of these definitions is to indicate the primary network perspective (the CA network)?

I got confused if this was intended to the 'primary domain' as in the 'naked' domain (i.e., example.com vs sub.example.com).

I tried to clarify this with the following suggestion, but potentially there is also a different definition we could use?
```suggestion
**Primary CAA Determination**: The primary perspective of which the CA has checked CAA records and determined its authority to issue a Certificate for the requested domain(s).

**Primary Domain Validation Determination**: The primary perspective of which the CA has determined Applicant authority and/or domain authorization or control using one of the methods described in Section 3.2.2.4 of this Policy.
```

Reply:
The intention was for these to refer to the primary perspective. I think the change looks good.

Reply:
Good point @vanbroup - I think these definitions have evolved over the course of requirements drafting and should be updated. Originally, these terms were intended to be broader than just "where" (i.e., the network perspective), but to also consider the outcome (i.e., determination made from that perspective re: validation was completed successfully and CAA records indicated the CA could proceed with issuance).

Given the current state of 3.2.2.9, I think there's an opportunity to simplify and collapse these terms into something like "Primary Network Perspective." 

I'll refactor this shortly.

Reply:
Updated in https://github.com/ryancdickson/staging/commit/e013f27c16be097217c37fe0922cff201bc54641